scripts/package/projectsToPack.ps1

```diff
@@ -3,10 +3,13 @@
 
 $ProjectsToPack = @(
     "Microsoft.DotNet.Cli.Utils",
+    "Microsoft.DotNet.Compiler.Common",
+    "Microsoft.DotNet.Files",
```

Comment:
We need to fix those dependencies. 

Microsoft.DotNet.Files shouldn't exist.
Microsoft.DotNet.Compiler.Common should only be used by the CLI.
Microsoft.DotNet.InternalAbstractions is an abomination that shouldn't have to leak to consumers. It's just for testing.